lib/hammer_cli_foreman/host.rb

```diff
@@ -52,12 +52,6 @@ def extend_data(host)
         host
       end
 
-      def get_parameters(host_id)
-        params = HammerCLIForeman.foreman_resource!(:parameters).call(:index, :host_id => host_id)
```

Comment:
This is a leftover that should have been removed when http://projects.theforeman.org/issues/15585 was fixed